src/encoded/static/components/app.js

```diff
@@ -790,7 +791,7 @@ export default class App extends React.PureComponent {
                     window.location.replace(targetHref + hashAppendage);
                 } else {
                     const [ old_path ] = ('' + window.location).split('#');
-                    window.location.assign(targetHref + hashAppendage);
+                    Sinon.stub(targetHref + hashAppendage);
```

Comment:
[Sinon.stub](https://sinonjs.org/releases/latest/stubs/) (and Sinon in general) is meant to be used within unit tests, not within the app code itself, I'm pretty sure..